src/Illuminate/Notifications/RoutesNotifications.php

```diff
@@ -20,6 +20,8 @@ public function notify($instance)
 
     /**
      * Get the notification routing information for the given driver.
+     * In case of an empty return value, the notification will
+     * not be sent.
```

Comment:
There no guarantee that this is the case. If you implement `ShouldQueue`, send() would return null because it would just dispatch to queue.